beta/src/pages/apis/dom-attributes/suppresscontenteditablewarning.md

```diff
@@ -0,0 +1,48 @@
+---
+title: suppressContentEditableWarning
+---
+
+<Intro>
+
+When an element with children is also marked as `contentEditable`, a warning is generated. You can suppress that warning by setting `suppressContentEditableWarning` attribute to `true`.
+
+</Intro>
+
+<Note>
+
+Don’t use `suppressContentEditableWarning` unless you are building a library like [Draft.js](https://draftjs.org/), that manages contentEditable manually.
+</Note>
+
+- [Usage](#usage)
+  - [Using `suppressContentEditableWarning` attribute](#using-style-suppressContentEditableWarning)
+
+## Usage {/*usage*/}
+
+### Using `suppressContentEditableWarning` attribute {/*using-style-suppressContentEditableWarning*/}
+
+In the example below, the `<div>` is editable because the `contentEditable` attribute is set to true, and you get a warning when this happens.
```

Comment:
```suggestion
In the example below, the `<div>` is editable because the `contentEditable` attribute is set to `true`, and you get a warning when this happens.
```